src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -456,6 +460,7 @@ private void modifyKubernetesPostDeployment() throws Exception {
   private static final String renderedConfigFileName = "RENDERED_testConfiguration.json";
   private static final String userJourneyResultsFileName = "RAWDATA_userJourneyResults.json";
   private static final String runSummaryFileName = "SUMMARY_testRun.json";
+  private static final String terraVersionFileName = "TERRA_componentVersion.json";
```

Comment:
nit: Can we rename the variable and file name from terra/TERRA -> environment/ENV_ since everything here is technically Terra-specific?

Reply:
Renamed the variable as suggested.